CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -957,6 +991,7 @@ public int DestroySwapChainResources(){
             // different position for our FPS.  After updating the uniform buffers we
             // need to update the descriptor sets that bind the uniform buffers as well.
             DestroyPipeline();
+            DestroyPipelineLayouts();
```

Comment:
Why did you add this?

Reply:
Because vkCreatePipelineLayout is called in CreatePipeline(), so previously we were re-creating the layouts without destroying them.

Reply:
Then that needs to change!  Pipeline layouts should be created only once, they are templates, I don't think there is any reason to tie them to the swapchain lifetime.

Reply:
I fixed Axes and FPS so they don't re-create any pipelines by making the viewport/scissor dynamic